### PR TITLE
Few socket updates related for Linux.

### PR DESCRIPTION
**Pull request by devnexen**

### CODE REVIEW

**Review by JohnTitor — APPROVED** [review]

**Comment by JohnTitor** [comment]
Seems emscripten doesn't have them.

**Comment by JohnTitor** [comment]
...Also Android doesn't have `MSG_ZEROCOPY`, it seems.

File: src/unix/linux_like/mod.rs [hunk 1]
```diff
@@ -775,6 +775,11 @@ pub const MSG_ERRQUEUE: ::c_int = 0x2000;
 pub const MSG_NOSIGNAL: ::c_int = 0x4000;
 pub const MSG_MORE: ::c_int = 0x8000;
 pub const MSG_WAITFORONE: ::c_int = 0x10000;
+cfg_if! {
+    if #[cfg(not(all(target_os = "emscripten", target_os = "android")))] {
```
Comment (JohnTitor) [inline src/unix/linux_like/mod.rs:779]: Could you just move them to src/unix/linux_like/linux/mod.rs to avoid cfg complexity?

Comment (devnexen) [reply src/unix/linux_like/mod.rs:779]: ah you re right

**Comment by JohnTitor** [comment]
Thanks! @bors r+

**Pull request merged**